### PR TITLE
Fixed dropdown overlap

### LINKED ISSUE #4026: More dropdown is overlapping with search
More dropdown is overlapping with search in navbar.
### Evidence / Screenshot (if possible)
Screenshot_2020-11-05 Welcome to Open Library Open Library

###Expected:  Can be shifted little down to remove over lapping.

#4012  @Yashs911 Please look into this also.

### PR DESCRIPTION
**Pull request by Yashs911**
Closes #4026 

### Screenshot

soln

### Stakeholders

@jamesachamp

### CODE REVIEW

**Comment by jamesachamp** [comment]
I'm having a difficult time moving my cursor to the drop-down component with this change.  Whenever the navigation component loses focus, the drop-down menu is hidden.

It may be better to revert this change and increase the z-index of the drop-down component.  There will still be overlap, but it will probably look better if the menu overlaps the search bar.

**Comment by Yashs911** [comment]
@jamesachamp I have made the required changes.

**Comment by Yashs911** [comment]
Hi, @jamesachamp if possible can you please this PR.

**Review by jdlrobson — CHANGES_REQUESTED** [review]
This doesn't seem to fix it.
.header-bar .search-component .search-bar-component is the same z-index.

Also using z-index-level-17 is an anti-pattern.

The reason this is happening is neither .search-component .navigation-component specifies a z-index. It should be possible to set the parent element .navigation-component to a higher z-index than search-component to get this. A z-index of 1 and 2 respectfully should help that. I recommend reading up on z-index stacking Order and stacking contexts.

**Comment by Yashs911** [comment]
@jdlrobson I have made the required changes

File: static/css/components/header-bar.less [hunk 1]
```diff
@@ -228,6 +229,7 @@
     -webkit-box-flex: 1;
     -ms-flex: 1;
     flex: 1;
+    z-index: 1;
```
Comment (jdlrobson) [inline static/css/components/header-bar.less:232]: Please use the appropriate variable in static/css/less/z-index.less - it helps us keep these under control.

Comment (Yashs911) [reply static/css/components/header-bar.less]: Done.

**Comment by Yashs911** [comment]
@jdlrobson Any Idea how to fix this? :)
`FAIL  static/build/page-plain.css: 20KB > maxSize 20KB (gzip) `

**Comment by jdlrobson** [comment]
>  FAIL static/build/page-plain.css: 20KB > maxSize 20KB (gzip) 
You've increased the bundlesize. It's only by a small amount so that's okay.

Update bundlesize.config.json  so it's 21KB.

**Comment by Yashs911** [comment]
@jdlrobson Done. Thank you.

**Comment by jdlrobson** [comment]
Merged. @Yashs911 I've amended your commit message again. Commit messages are useful for future engineers when debugging **why,** we did, and **what** we did, without having to parse the diff so please be sure to make them as descriptive as possible and reference the issue.

**Pull request merged**